src/lib/models/keyframe.js

```diff
@@ -1,62 +1,193 @@
 'use strict';
 
+const _     = require("lodash")
```

Comment:
Lots of spaces?

Reply:
Ah, leftover from when where were more requires I think.